src/object/srv_obj_migrate.c

```diff
@@ -1766,6 +1860,129 @@ migrate_one_destroy(struct migrate_one *mrone)
 	D_FREE(mrone);
 }
 
+enum {
+	OBJ_ULT = 1,
+	DKEY_ULT = 2,
+};
+
+/* Check if there are enough resource for the migration to proceed. */
+static int
+migrate_system_enter(struct migrate_pool_tls *tls, int tgt_idx)
```

Comment:
"int tgt_idx" better to be "uint32_t tgt_idx"

Reply:
yeah, but tgt_idx is the index within one engine, so it is barely more than 16.